src/worker.ts

```diff
@@ -123,53 +129,55 @@ async function handleReceiveFileData({id}: RPCMessage): Promise<void> {
     }
 }
 
-onmessage = async function (event) {
-    if (!isRPCMessage(event.data)) {
-        throw new Error(`unexpected event: ${JSON.stringify(event, null, '  ')}`);
-    }
-
-    // NB: unregister worker message handler.
-    //  (use message channel port instead)
-    onmessage = () => {
-        // NB: noop
-    };
+if (!browserIsProbablySafari) {
```

Comment:
@reviewer the diff here is adjusting indentation between (new) lines 133-182 and wrapping everything that would run on worker startup in the conditional expression on this line.